restructuredtext_lint/cli.py

```diff
@@ -31,32 +47,39 @@ def _main(filepaths, format='text', stream=sys.stdout, encoding=None):
                     stream.write('{err.type} {err.source}:{err.line} {err.message}\n'.format(err=err))
             elif format == 'json':
                 error_dicts.extend({
-                    'line': error.line,
-                    'source': error.source,
-                    'level': error.level,
-                    'type': error.type,
-                    'message': error.message,
-                    'full_message': error.full_message,
-                } for error in file_errors)
+                    'line': err.line,
+                    'source': err.source,
+                    'level': err.level,
+                    'type': err.type,
+                    'message': err.message,
+                    'full_message': err.full_message,
+                } for err in file_errors)
```

Comment:
We should probably leave this section alone. It will be a whole other PR to rename all `error` to `err` and it doesn't help us much development-wise =/